src/store-event-bus.ts

```diff
@@ -1,17 +1,21 @@
 import { Injectable } from '@nestjs/common';
-import { IEvent, IEventBus } from '@nestjs/cqrs/dist/interfaces';
+import { IEvent } from '@nestjs/cqrs/dist/interfaces';
 import { EventStore } from './eventstore';
-import { StorableEvent } from './interfaces/storable-event';
-import { ViewEventBus } from './view/view-event-bus';
+import { StorableEvent } from './interfaces';
+import { IAppKnitEventBus, ViewEventBus } from './view';
+import { AggregateRootAsync } from './aggregate-root-async';
 
 @Injectable()
-export class StoreEventBus implements IEventBus {
+export class StoreEventBus implements IAppKnitEventBus {
```

Comment:
use IEventBus as we can contribute back to original repo

Reply:
Renamed to IExtendedEventBus